README.md

```diff
@@ -1,6 +1,188 @@
 [![official project](http://jb.gg/badges/official.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub)
 [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-# [Compose Multiplatform](https://github.com/JetBrains/compose-multiplatform) Desktop Application
+# [Compose Multiplatform](https://github.com/JetBrains/compose-multiplatform) desktop application
 
-- `./gradlew run` - run application
-- `./gradlew package` - package native distribution into `build/compose/binaries`
+> **Note**
```

Comment:
We don't want feedback on the desktop target. We want for iOS and web only. Let's remove this note. 
Or rewrite:
> If you have any questions about Compose for Desktop, ask them in the [#compose-desktop](https://kotlinlang.slack.com/archives/C01D6HTPATV/p1678887144652589) Slack channel. ...